docs/examples/infobutton/intermixedBlocks.js

```diff
@@ -0,0 +1,16 @@
+// @flow strict
```

Comment:
I take it these are placeholders?

Reply:
It wasn't my intention, what do you suggest to these fields?

Reply:
Ah ok. We need visuals from @ponori for you to build, otherwise we'll need to omit the visuals for this section (or omit the section altogether)

Reply:
Here's the final example for Do:
![image](https://user-images.githubusercontent.com/50343812/216158063-8352ac97-5da2-414c-8434-aa9b01151c96.png)

And here's the final example for Don't:
![image](https://user-images.githubusercontent.com/50343812/216158137-e037a13d-9882-46ff-b9f6-e3e81cd32c09.png)

With each popover having the following messages:
* page visits: "Number of times people visited your website after seeing your content on Pinterest"
* add-to-carts: "Number of times people added your products to their carts after interacting with your Pins"
* checkouts: "Number of checkouts stemming from your Pins and ads"